api_breaking_changes.rdoc

```diff
@@ -0,0 +1,79 @@
+== Breaking changes in the major releases
+
+=== 1.0.0
+
+To use the client use the <code>require 'hawkular/hawkular_client'</code> instead of <code>require 'hawkular_all'</code>
```

Comment:
+1 to highlight it in the main changes doc, this is not an api change, but a naming change.